woocommerce/woocommerce-android
# Show revenue card in analytics hub

### Closes
This pull request is part of [5237](https://github.com/woocommerce/woocommerce-android/issues/5237)

### Description

This pull requests shows the revenue card in the new analytics hub section of the app. It has 3 states:

1. Loading state shows an skeleton while data is being fetch from the server
2. Error shows a message
3. Data shows total sales, net sales and the difference with the same data of previous period  

### Testing instructions

1. Compile app in debug mode
2. Open it
3. Clicks on analytics section
4. Revenue card is shown
5. Change the selected period by clicking on the calendar icon above
6. Revenue card should show the expected data

### Images/gif

<img width="317" alt="Screenshot 2021-12-06 at 19 58 07" src="https://user-images.githubusercontent.com/572601/144905701-7bba3da0-68ba-4bf9-aae2-d3eae26cb5a5.png">
<img width="389" alt="Screenshot 2021-12-06 at 19 56 46" src="https://user-images.githubusercontent.com/572601/144905706-fab3c51e-b226-480e-a89f-c590da0a08ba.png">
